scripts/spack/configs/docker/ubuntu20/spack.yaml

```diff
@@ -9,7 +9,7 @@ spack:
     build_stage::
     - $spack/../build_stage
 
-  compilers::
```

Comment:
Unless this was causing an issue we should still have the double colon `::`. It's a spack thing - it pretty much means "only use the compilers below".